Dockerfile

```diff
@@ -0,0 +1,15 @@
+# golang:1.17 is based on debian:11, this is important to ensure we have libc compatibility for the copied binary
+
+FROM docker.io/library/golang:1.17 as builder
+ENV CGO_ENABLED=1
```

Comment:
I don't know how, but the size of the mantle binaries built with `CGO_ENABLED=1` or `CGO_ENABLED=0` is essentially the same, so we could also get rid of this restriction.

Reply:
I'm happy to try out disabling cgo but in a follow up PR where we make sure that all works compared to the current state